admissions/static/admissions/js/bootstrap.min.js

```diff
@@ -0,0 +1,6 @@
+/*!
```

Comment:
these files should be in static/admissions/js/ so that they can be accessed by other apps as well